tests/conftest.py

```diff
@@ -991,6 +992,18 @@ def pytest_runtest_makereport(item, call):
     setattr(item, "rep_" + rep.when, rep)
 
 
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_call(item):
```

Comment:
Nice hack. Can you add more docstring to explain the motivation and ideas under these code changes?

Reply:
With this change in place, probably you'll need to do some cleanup on existing test scripts?

Reply:
Sure, will add more docstring and do cleanup